Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    public class BalanceStats
+    {
+        private readonly string[] _peopleAndBalances;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_peopleAndBalances' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5054323)